charts/opentelemetry-collector/README.md

```diff
@@ -104,6 +104,29 @@ presets:
 The way this feature works is it adds a `filelog` receiver on the `logs` pipeline. This receiver is preconfigured
 to read the files where Kubernetes container runtime writes all containers' console output to.
 
+#### Log collection for a subset of pods or containers
+
+The `logsCollection` preset will by default ingest the logs of all kubernetes containers.
+This is achieved by using an include path of `/var/log/pods/*/*/*.log` for the `filelog`receiver.
+
+To limit the import to a certain subset of pods or containers, the `filelog`
+receivers `include` list can be overwritten by supplying explicit configuration.
+
+E.g. The following configuration would only import logs for pods within namespace `example-namespace`:
+
+```yaml
+mode: daemonset
+
+presets:
+  logsCollection:
+    enabled: true
+config:
+  receivers:
+    filelog:
+      include:
+        - /var/log/pods/example-namespace_*/*/*.log
+```
+
 #### :warning: Warning: Risk of looping the exported logs back into the receiver, causing "log explosion"
```

Comment:
This warning should stay close to the section before the addition

Reply:
@BeElectronicSakshi the warning will needs to be moved back up